include/cantera/thermo/ThermoPhase.h

```diff
@@ -109,6 +109,11 @@ class ThermoPhase : public Phase
         return "ThermoPhase";
     }
 
+    //! String indicating the phase (liquid, vapor, etc.) of the phase
+    virtual std::string getPhase() const {
+        throw NotImplementedError("ThermoPhase::getPhase");
+    }
```

Comment:
Should this method just be called `phase()`? Normally in the C++ interface, we use `getFoo` for a method that uses one of its arguments as an output, while the simple property getters are just the property name, e.g. `type()` and `temperature()`.

Reply:
I don't like `phase()` (I don't like `getPhase()` either) because we overload the word "phase" to mean so many things. I experimented with calling this idea "mechanical phase" (or maybe "thermo-mechanical phase") in prose, and I think that makes some more sense (albeit a bit unwieldy), but that won't work for a method name. I've also thought of `phaseType()` (too similar to `type()`), `phaseName()` (too similar to `name()`), and `phaseOfMatter()`. I like the last one the best, although it's maybe too long... Ideas appreciated!

Reply:
Agreed, the many meanings of 'phase' are a challenge. `phaseOfMatter` actually sounds pretty good to me, and I don't think it's overly verbose compared to many other method names that we have like `getPartialMolarEnthalpies`.